azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/compute/options/AzureTemplateOptions.java

```diff
@@ -76,12 +83,50 @@ public AzureTemplateOptions availabilitySet(String availabilitySetName) {
       return this;
    }
 
+   public AzureTemplateOptions dataDisks(List<Map<String, String>> dataDisksAsMap) {
```

Comment:
I think it'd simplify the consumption from downstream projects like Apache Brooklyn. wdyt?